### PR TITLE
Add angle65 

### PR DESCRIPTION
**Pull request by Vertex-kb**
## Description
Add angle65 

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/19986

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/vertex/angle65/angle65.JSON [hunk 1]
```diff
@@ -0,0 +1,193 @@
+{
+  "name": "ANGLE65",
+  "vendorId": "0x9954",
+  "productId": "0x9970",
```
Comment (Xelus22) [inline v3/vertex/angle65/angle65.JSON:4]: VID and PID incorrect with QMK ones

File: src/vertex/angle65/angle65.JSON [hunk 2]
```diff
@@ -0,0 +1,194 @@
+{
+  "name": "ANGLE65",
+  "vendorId": "0x9954",
+  "productId": "0x9970",
```
Comment (Xelus22) [inline src/vertex/angle65/angle65.JSON:4]: Remove V2 definitions as it is not needed (src folder one)

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/vertex/angle65/angle65.JSON (hunk 1 above)
Comment (Vertex-kb) [reply v3/vertex/angle65/angle65.JSON:4]: I have modified duplicate VID/PID https://github.com/qmk/qmk_firmware/pull/20401

File: src/vertex/angle65/angle65.JSON (hunk 2 above)
Comment (Vertex-kb) [reply src/vertex/angle65/angle65.JSON:4]: Some PCB firmware uses the previous version of QMK, V2 is still necessary. Thanks

**Pull request closed without merging**